assignments/suchomel/Dartboard_heuristic_optimization.py

```diff
@@ -0,0 +1,542 @@
+
```

Comment:
What is the reason for keeping a copy (I presume) of `Dartboard_heuristic_optimization.ipynb` in the `sources` folder?

Reply:
A mistake, I just wanted to be sure not to lose the computed results, but this is not the way...